example_sns_init.yaml

```diff
@@ -0,0 +1,117 @@
+# You should make a copy of this file, name it sns_init.yaml, and edit it to
+# your taste.
+#
+# All principal IDs should almost certainly be changed.
```

Comment:
That makes sense for some, but not all principal IDs should be that of the current dfx identity. E.g. the dapp canisters. I can replace the ones that should be though. I think that includes just fallback principal, and one of the neurons.

Reply:
> and one of the neurons.

I wouldn't keep neurons with arbitrary principals in the yaml file. How about having the two neurons both controlled by the current DFX identity?

Reply:
I want to illustrate that each member of the team would get one or more neuron, not just the person who happens to be performing SNS-ification.

Reply:
Sure, but note that might require voting on proposals already in the pre-swap phase and thus we'd need to extend the basic scenario. Another option: sns CLI tool produces an example yaml file with multiple dummy identities, but the example yaml file checked in sns-testing is as simple as possible so that no voting is required in the pre-swap phase.

Reply:
Ok, to address my concern, I replaced the second neuron with a comment that approximately says "IRL, you would have more neurons here, assuming your team has more than one person, and you are not trying to screw them.".